runbotics/runbotics-common/src/RunboticModdleDescriptor.ts

```diff
@@ -33,6 +33,12 @@ export const RunboticModdleDescriptor = {
                     type: "Boolean",
                     default: false,
                 },
+                {
+                    name: "processOutput",
+                    isAttr: true,
+                    type: "Boolean",
+                    default: false,
+                },
```

Comment:
what is it for?

Reply:
it was redundant

Reply:
I was wrong, this is necessary to be able persist changes while updating process definition